corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -205,3 +97,141 @@ def handle(self, **options):
         resource_versions["hqwebapp/js/resource_versions.js"] = self.get_hash(filename)
 
         self.overwrite_resources(resource_versions)
+
+
+def _confirm_or_exit():
+    proc = subprocess.Popen(["git", "diff-files", "--ignore-submodules", "--name-only"],
+                            stdout=subprocess.PIPE)
```

Comment:
To be consistent with the first commit, should this use `with subprocess.Popen(...) as proc:`?

Reply:
I agree. I was planning to do that later -- only the git subprocesses were resulting in warnings -- so I figured that I shouldn't change any functionality in the refactor commits, and make changes afterwards.